azure_arc_k8s_jumpstart/cluster_api/capi_keyvault_extension/capi_keyvault_k8s_extension.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# <--- Change the following environment variables according to your Azure service principal name --->
+export appId='<Your Azure service principal name>'
+export password='<Your Azure service principal password>'
+export tenantId='<Your Azure tenant ID>'
+export keyVaultResourceGroup='<KeyVault Resource Group name>'
+export keyVaultLocation='<Key Vault Location>'
+export resourceGroup='<Azure Arc CLuster Resource Group name>'
+export arcClusterName='<Azure Arc Cluster Name>'
+export namespace='hello-arc'
+export k8sExtensionName='sscsi'
+export keyVaultName=secret-store-$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1)
+
+# Installing Helm 3
+echo "Installing Helm 3"
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
```

Comment:
what happens if az or helm 3 are already installed?

Reply:
It will continue to the next step with the message "Helm v3.x.x. is already latest"